### PR TITLE
Duplicates removal

### PR DESCRIPTION
**Pull request by PA-Rweins**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Fixed some entities with duplicated ids

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Cymulate/ReleaseNotes/2_0_6.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Playbooks
+##### Cymulate Immediate Threats
+- Updated version needed.
```
Comment (yaakovi) [inline Packs/Cymulate/ReleaseNotes/2_0_6.md:4]:
```suggestion
- Maintenance and stability enhancements.
```

File: Packs/XMCyber/ReleaseNotes/1_0_9.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+
+#### Incident Fields
+- Removed **# Critical Assets at Risk**
```
Comment (yaakovi) [inline Packs/XMCyber/ReleaseNotes/1_0_9.md:3]:
```suggestion
- Updated **# Critical Assets at Risk**
```

File: Packs/Cymulate/TestPlaybooks/playbook-CymulateTest.yml [hunk 3]
```diff
@@ -1,6 +1,6 @@
-id: Cymulate Immediate Threats
+id: Cymulate V1 Test
```
Comment (yaakovi) [inline Packs/Cymulate/TestPlaybooks/playbook-CymulateTest.yml:1]: don't you need to update `conf.json`?

**Review by yaakovi — APPROVED** [review]

File: Packs/Cymulate/ReleaseNotes/2_0_6.md (hunk 1 above)
Comment (PA-Rweins) [reply Packs/Cymulate/ReleaseNotes/2_0_6.md:4]: fixed

File: Packs/XMCyber/ReleaseNotes/1_0_9.md (hunk 2 above)
Comment (PA-Rweins) [reply Packs/XMCyber/ReleaseNotes/1_0_9.md:3]: fixed

File: Packs/Cymulate/TestPlaybooks/playbook-CymulateTest.yml (hunk 3 above)
Comment (PA-Rweins) [reply Packs/Cymulate/TestPlaybooks/playbook-CymulateTest.yml:1]: You're right, fixed it.

**Pull request merged**